packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -270,6 +271,15 @@ function _DevtoolsView(props: _DevtoolsViewProps): React.ReactElement {
 				setContainers(message.data.containers);
 				return true;
 			},
+			[DevtoolsDisposed.MessageType]: async (untypedMessage) => {
+				// eslint-disable-next-line unicorn/no-useless-undefined
+				setMenuSelection(undefined);
+				// eslint-disable-next-line unicorn/no-useless-undefined
+				setContainers(undefined);
+				// Require latest feature state to ensure we aren't displaying stale data
+				messageRelay.postMessage(getSupportedFeaturesMessage);
```

Comment:
Nit: I think where you had this before (in the parent component, and resetting the `supportedFeatures` state was probably a better place for this.

I'm guessing we were seeing stale `containers` and `menuSelection` state when we were doing it that way? In that case, my preference would be to put this back the way it was, and add a `useEffect` block in this component to reset those pieces of state when `supportedFeatures` changes.